packages/flutter/lib/widgets.dart

```diff
@@ -131,6 +131,7 @@ export 'src/widgets/scrollbar.dart';
 export 'src/widgets/selectable_region.dart';
 export 'src/widgets/selection_container.dart';
 export 'src/widgets/semantics_debugger.dart';
+export 'src/widgets/sensitive_content.dart';
```

Comment:
If we want to go with visibleForTest we should not export the file here. and import the file directly in the test with a todo to fix it once the widget is ready for consumption.

and we should only ever do this if this is temporary which i assume it is.

Reply:
Ok I'll make these changes, address Reid's review, and run this past @justinmc since you mentioned this goes against the style guide.

Reply:
This sounds good to me per https://github.com/flutter/flutter/issues/164820#issuecomment-2725474314.